corehq/apps/data_dictionary/views.py

```diff
@@ -67,10 +68,17 @@ def data_dictionary_json(request, domain, case_type_name=None):
             domain)
     if case_type_name:
         queryset = queryset.filter(name=case_type_name)
+
+    case_type_app_count = get_case_type_app_count(domain)
     for case_type in queryset:
+        app_count = 0
+        if case_type.name in case_type_app_count:
+            app_count = case_type_app_count[case_type.name]
```

Comment:
nit: you can rewrite this as
```suggestion
        app_count = case_type_app_count.get(case_type.name, 0)
```